source/backend/apimodels/Models/Concepts/DispositionFile/DispositionSalePurchaserModel.cs

```diff
@@ -15,7 +15,7 @@ public class DispositionSalePurchaserModel : BaseConcurrentModel
         /// <summary>
         /// Parent Disposition Sale.
         /// </summary>
-        public long DispositionSaleId { get; set; }
+        public long? DispositionSaleId { get; set; }
```

Comment:
same as last question

Reply:
For new entities I am passing null in the model and letting EF to resolve the primary key to be generated.